### PR TITLE
Add to add-layer [op-51]

### PR DESCRIPTION
**Pull request by Rytiggy**

### CODE REVIEW

File: app/components/add-layer/component.js [hunk 1]
```diff
@@ -221,6 +221,112 @@ export default Ember.Component.extend({
                 }
             };
             break;
+                case 'image':
+                item = {
+            "sectionHeader": "Image",
```
Comment (caneruguz) [inline app/components/add-layer/component.js]: Can you fix the indents?

File: app/components/add-layer/template.hbs [hunk 2]
```diff
@@ -1,37 +1,23 @@
...
-
-            <div class="col-sm-2">
-                <button class="btn btn-default btn-block" {{action 'addLayer' 'advanced'}}>{{fa-icon "paint-brush"}} <br /> Add Advanced</button>
-            </div>
-
-            <div class="col-sm-2">
-            <button class="btn btn-default btn-block" {{action 'addLayer' 'navigation'}}>{{fa-icon "paint-brush"}} <br /> Add navigation</button>
+        <div class="col-sm-12 text-center">
+                <div class="layer-links">
+                    <a class="fa fa-wikipedia-w layer-link" href="#" title="Add wiki" {{action 'addLayer' 'wiki'}}></a>
```
Comment (caneruguz) [inline app/components/add-layer/template.hbs]: These shouldn't be a tags, revised button is better

File: app/components/add-layer/style.scss [hunk 3]
```diff
@@ -46,3 +55,60 @@
...
+    content: attr(title);
+    width: 100%;
+    text-align: center;
+    left: 0;
+    color:white;
+    bottom: ($size * -3);
+    font-size: .35em;
+    position: absolute;
+    transition: .3s;
+    font-family: 'Alegreya Sans SC', sans-serif;
```
Comment (caneruguz) [inline app/components/add-layer/style.scss]: This looks like a new font but where is it loading? 

Also can we not use a new font? Are these icon fonts?

File: app/components/add-layer/component.js [hunk 4]
```diff
@@ -221,6 +221,112 @@ export default Ember.Component.extend({
                 }
             };
             break;
+                case 'image':
```
Comment (caneruguz) [inline app/components/add-layer/component.js]: There is already a ```case 'image'``` above, isn't there?

Comment (Rytiggy) [inline app/components/add-layer/component.js]: copy and paste error

**Comment by Rytiggy** [comment]
screen shot 2017-07-31 at 4 55 49 pm

**Pull request merged**